docs/settings/breakpoint-settings.md

```diff
@@ -1,50 +1,49 @@
 ---
-title: Breakpoints settings
-table_of_contents: true
+layout: default
 ---
 
 ## Breakpoints
 
 Vanilla uses four main breakpoints for screen sizes, below you can see the setting, default width and device examples at which the content will scale.
 
-Setting | Default value | Device example
-------------- | ------------- | -------------
-`$breakpoint-x-small`                | `460px`              | Mobile
-`$breakpoint-small`                  | `620px`              | Mobile
-`$breakpoint-medium`                 | `768px`              | Tablet
-`$breakpoint-large`                  | `1030px`             | Desktop
-`$breakpoint-navigation-threshold`   | `$breakpoint-medium` | Tablet
-`$breakpoint-heading-threshold`      | `$breakpoint-medium` | Tablet
+| Setting                            | Default value        | Device example |
```

Comment:
:)

Reply:
This wasn't even me. This happened automatically.